src/controls/adaptiveCardDesignerHost/AdaptiveCardDesigner.Helpers.ts

```diff
@@ -0,0 +1,59 @@
+import { Guid } from '@microsoft/sp-core-library';
```

Comment:
Could you please add some comments for these helpers?
Like why do we need that? what's the current assumption on Toolbar HTML?

Reply:
I used this to have a unique Id to register the various controls in the toolbar (required by the Adaptive Card Designer component of the original SDK). Do you suggest an alternative way to get a Unique ID?

Reply:
@fabiofranzini - no, I'm just asking to add comments to the code :)

Reply:
Ok, I understand what you mean... These helpers are for internal use only and make no sense "externally". Do you think it is better to remove the exports from these functions and move them directly into the "AdaptiveCardDesigner.tsx" file?

Reply:
For me most importantly is to have comments on what the helpers do and why. In that case if someone else is going to make changes in future - it'll be easier.